test/integration/planets.test.js

```diff
@@ -0,0 +1,16 @@
+import test from 'ava'
+import boot from '../../src/boot'
+import visit from '../helpers/visit'
+
+test.cb('shows a list of planets on the homepage', assert => {
+  const homepage = visit('/')
+
+  setTimeout(function () {
+    const planets = homepage.querySelectorAll('li')
+
+    assert.is(planets[0].textContent, 'Mercury')
+    assert.is(planets[8].textContent, 'Pluto')
+
+    assert.end()
+  }, 250)
```

Comment:
I'm playing around with better integration testing (distinct from acceptance testing). `visit` mounts the app to a `div` at a given path. Basically the same as rendering a page.